tests/testdata/run/permissions_prompt_allow_all_2.ts

```diff
@@ -5,4 +5,4 @@ Deno.loadavg();
 Deno.hostname();
 
 Deno.cwd();
-Deno.lstatSync(new URL("../", import.meta.url));
+Deno.lstatSync(new URL("..", import.meta.url));
```

Comment:
If doing `../` then on Windows it will have no trailing slash, but on Linux it does. It doesn't matter too much except for in the output of tests, so I think doing ".." might make it output the same on all operating systems and pass the CI (hopefully).